File: data/db/src/main/kotlin/red/razvan/restcountries/data/db/Currency.kt
// Copyright 2025 Răzvan Roșu
// SPDX-License-Identifier: Apache-2.0

package red.razvan.restcountries.data.db

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(
  tableName = "currency",
)
data class Currency(
  @PrimaryKey
  val id: CurrencyId,
  val name: String,
  val symbol: String,
)

File: ios-app/rest-countries/Sources/Models/Currency.swift
//
//  Currency.swift
//  RestCountries
//
//  Created by Răzvan Roşu on 13/08/25.
//

import Foundation

public struct Currency: Hashable, Identifiable {
  public let id: CurrencyId
  public let name: String
  public let symbol: String
}
